feature/system/gnmi/cliorigin/mixed_oc_cli_origin_support_test/README.md

```diff
@@ -0,0 +1,55 @@
+# gNMI-1.1: cli Origin
+
+## Summary
+
+Ensure that both CLI and OC configuration can be pushed to the device at the
+same time
+
+## Procedure
```

Comment:
I'm going to push an edit with slight rewording. If you're okay with the new wording, can you copy each procedure step to a code comment at the appropriate places?

Reply:
Done